deployment/local/util.ts

```diff
@@ -0,0 +1,3 @@
+export function isLocalStack() {
+  return process.env.CDK_DEFAULT_ACCOUNT === "000000000000";
+}
```

Comment:
There's a chance this could be:
export const isLocalStack = process.env.CDK_DEFAULT_ACCOUNT === "000000000000";
but again I'm having to look really hard to find any improvements